specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-11-01/virtualMachine.json

```diff
@@ -3586,6 +3590,36 @@
         }
       },
       "description": "Describes the properties of the last installed patch summary."
+    },
+    "Placement": {
+      "properties": {
+        "zonePlacementPolicy": {
+          "type": "string",
+          "description": "Specifies the policy for virtual machine's placement in availability zone. <br>Possible values are:<br><br>**Any** - An availability zone will be automatically picked by system as part of virtual machine creation.",
+          "enum": [
+            "Any"
+          ],
+          "x-ms-enum": {
+            "name": "ZonePlacementPolicyType",
+            "modelAsString": true
+          }
+        },
+        "includeZones": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "This property supplements the 'zonePlacementPolicy' property.<br>If 'zonePlacementPolicy' is set to 'Any', availability zone selected by the system must be present in the list of availability zones passed with 'includeZones'. If 'includeZones' is not provided, all availability zones in region will be considered for selection."
```

Comment:
we don't allow <br> in the descriptions anymore because it does not parse the format correctly in certain platforms

Reply:
Removed all 'br's, 
Isn't there an alternative to this, because the documentation preview looks a bit ugly